src/data.rs

```diff
@@ -57,6 +61,10 @@ impl<T> TupleStruct<T> {
             fields: self.fields.clone().into_iter(),
         }
     }
+
+    pub fn attrs(&self) -> &[AttributeWrapper] {
```

Comment:
These should be returning &[Attribute], like we have already on reflect::Field. I don't want an AttributeWrapper type in the public API.

Reply:
I'm not sure that I will be able to construct a slice and return it, I'll try to return `impl Iterator<Item = syn::Attribute>`.